migrations/20220117025847_email_data.up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TYPE valid_status AS ENUM ('pending', 'running', 'completed', 'stopped');
+CREATE TABLE blk_vrfy_job (
+    id SERIAL PRIMARY KEY,
+    job_uuid UUID,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    attempt_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```

Comment:
What is attempt_at?

Reply:
This field is supposed to hold the timestamp when the bulk job started getting processed. But in the current approach to submit each email as a separate task to the queue this is no longer required.